build.sbt

```diff
@@ -0,0 +1,58 @@
+
+import TestPhases._
+import play.routes.compiler.InjectedRoutesGenerator
+import play.sbt.routes.RoutesKeys.routesGenerator
+import sbt.Keys._
+import sbt.{Def, inConfig, _}
+import uk.gov.hmrc.DefaultBuildSettings.{addTestReportOption, defaultSettings, scalaSettings}
+import uk.gov.hmrc.{SbtArtifactory, SbtAutoBuildPlugin, _}
+import uk.gov.hmrc.sbtdistributables.SbtDistributablesPlugin
+import uk.gov.hmrc.sbtdistributables.SbtDistributablesPlugin.publishingSettings
+import uk.gov.hmrc.versioning.SbtGitVersioning
+import uk.gov.hmrc.versioning.SbtGitVersioning.autoImport.majorVersion
+
+val appName = "ated-subscription-frontend"
+
+lazy val appDependencies: Seq[ModuleID] = AppDependencies()
+lazy val plugins: Seq[Plugins] = Seq(play.sbt.PlayScala)
+lazy val playSettings: Seq[Setting[_]] = Seq.empty
+
+lazy val scoverageSettings: Seq[Def.Setting[_ >: String with Double with Boolean]] = {
+  import scoverage.ScoverageKeys
+  Seq(
+    ScoverageKeys.coverageExcludedPackages := "<empty>;Reverse.*;views.html.*;views.*;app.Routes.*;prod.*;uk.gov.hmrc.*;testOnlyDoNotUseInAppConf.*;forms.*;config.*;",
+    ScoverageKeys.coverageMinimum := 90,
+    ScoverageKeys.coverageFailOnMinimum := true,
+    ScoverageKeys.coverageHighlighting := true
+  )
+}
+
+lazy val microservice = Project(appName, file("."))
+  .enablePlugins(Seq(play.sbt.PlayScala, SbtAutoBuildPlugin, SbtGitVersioning, SbtDistributablesPlugin, SbtArtifactory) ++ plugins: _*)
+  .settings(playSettings: _*)
+  .settings( majorVersion := 2 )
+  .configs(IntegrationTest)
+  .settings(scalaSettings: _*)
+  .settings(publishingSettings: _*)
+  .settings(defaultSettings(): _*)
+  .settings(scalaVersion := "2.12.11")
+  .settings(playSettings ++ scoverageSettings: _*)
+  .settings(
+    addTestReportOption(IntegrationTest, "int-test-reports"),
+    inConfig(IntegrationTest)(Defaults.itSettings),
+    libraryDependencies ++= appDependencies,
+    retrieveManaged := true,
+    evictionWarningOptions in update := EvictionWarningOptions.default.withWarnScalaVersionEviction(false),
+    routesGenerator := InjectedRoutesGenerator,
+    Keys.fork                  in Test            := true,
+    Keys.fork                  in IntegrationTest :=  false,
+    unmanagedSourceDirectories in IntegrationTest :=  (baseDirectory in IntegrationTest)(base => Seq(base / "it")).value,
+    parallelExecution in IntegrationTest := false
+  )
+  .settings(inConfig(TemplateTest)(Defaults.testSettings): _*)
+  .settings(inConfig(TemplateItTest)(Defaults.itSettings): _*)
+  .configs(IntegrationTest)
+  .settings(
+    resolvers += Resolver.bintrayRepo("hmrc", "releases"),
+    resolvers += Resolver.jcenterRepo
+  )
```

Comment:
Need to add entry for .disablePlugins(JUnitXmlReportPlugin) to sbt file otherwise the build will fail

Reply:
Done